performance.rst

```diff
@@ -96,8 +96,7 @@ value too using the ``realpath_cache_ttl`` option:
 Use Composer's Class Map Functionality
 --------------------------------------
 
-By default, the Symfony Standard Edition uses Composer's autoloader
-in the `autoload.php`_ file. This autoloader is easy to use, as it will
```

Comment:
Does not exist anymore  since https://github.com/symfony/symfony-standard/pull/1056